### PR TITLE
Add test for null-coalescing operator on nullable tuples with names

### PR DESCRIPTION
**Pull request by jcouv**
Test-only change.
Verifies https://github.com/dotnet/roslyn/issues/16825

@VSadov @dotnet/roslyn-compiler for review

### CODE REVIEW

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs [hunk 1]
```diff
@@ -16882,6 +16882,86 @@ public void M()
...
+        (int a, int c)? nac = (1, 3);
+
+        var x1 = nab ?? nac; // (a, b)?
+        var x2 = nab ?? nac.Value; // (a, b)
+        var x3 = new C() ?? nac; // C
+        var x4 = new D() ?? nac; // (a, c)?
+
+        var x5 = nab != null ? nab : nac; // (a, )?
+
+        var x6 = nab ?? (a: 1, c: 3); // (a, b)
```
Comment (VSadov) [inline src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs]: Try adding
`var x6double = nab ?? (a: 1.1, c: 3); // (a, c) `

There should be no warnings in this case.

**Comment by VSadov** [comment]
LGTM
one scenario suggestion

**Comment by AlekseyTs** [comment]
@jcouv Should we add similar test(s) for VB?

**Comment by jcouv** [comment]
Yes, Will do

**Pull request merged**